taichi/codegen/spirv/spirv_codegen.cpp

```diff
@@ -2156,7 +2156,7 @@ class TaskCodegen : public IRVisitor {
 
       // continue
       spirv::Value total_invocs = ir_->cast(
-          ir_->i32_type(),
+          ir_->u32_type(),
```

Comment:
? how it passed CI

Reply:
The SPIR-V builder does absolutely no type checking at all...